src/wasm-lib/kcl/src/executor.rs

```diff
@@ -2710,6 +2719,44 @@ const bracket = startSketchOn('XY')
         parse_execute(ast).await.unwrap();
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_unary_operator_not() {
+        let ast = r#"
+fn returnTrue = () => { return !false }
+const t = true
+const f = false
+let notTrue = !t
+let notFalse = !f
+let c = !!true
+let d = !returnTrue()
+// Yup, this is null.
+let myNull = 0 / 0
+let notNull = !myNull
+let notZero = !0
```

Comment:
theres the new assert functions to you could use and parse and execute as tests :)